espnet2/bin/s2t_inference.py

```diff
@@ -1029,6 +1066,32 @@ def get_parser():
         "If not given, refers from the training args",
     )
 
+    group = parser.add_argument_group("Semi-AR related")
+    group.add_argument(
+        "--partial_ar",
+        action="store_true",
```

Comment:
`store_true` is a bit tricky. Why not use `type=str2bool`? Please check the other boolean arguments.